website/mails/mails.py

```diff
@@ -177,7 +177,8 @@ def get_english_article(word):
 )
 CONFIRM_EMAIL_PREPRINTS = lambda name, provider: Mail(
     'confirm_preprints_{}'.format(name),
-    subject='Open Science Framework Account Verification, {} Preprints Service'.format(provider)
+    subject='Open Science Framework Account Verification, {} Preprints Service'.format(provider) if provider != 'Thesis Commons'
```

Comment:
I hate special casing for Thesis Commons, but I don't have a better suggestion right now, so it'll likely stay until we have to rip it out later.

Reply:
Maybe another field for the model like a "service_full_name" or "provider_full_name". But yes there's no better way if we want a hotfix and avoid migrations.

Reply:
How about just removing "Preprints Service":

> Open Science Framework Account Verification, Thesis Commons

> Open Science Framework Account Verification, SocArxiv

@nicipfeiffer

Reply:
For now, we can use a more robust check:

```python
...Preprints Service'.format(provider) if preprint_word != 'preprint' else ...
```

Reply:
It _could_ be the plural word unless it's Thesis Commons, but yeah, this is going to have to be re-worked soon.

Reply:
Talked to Nici on Flowdock, it is fine to remove "Preprints Service" in the subject, so the special casing can be removed.